index.d.ts

```diff
@@ -4,8 +4,8 @@ import { Instance, Options, GroupOptions } from 'tippy.js'
 type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
 
 export interface TippyProps extends Omit<Options, 'content'> {
-  content: React.ReactElement<any> | string
-  children: React.ReactElement<any>
+  content: React.ReactChild | React.ReactChild[]
+  children: React.ReactChild | React.ReactChild[]
```

Comment:
`<Tippy />` can't take an array of children though.

Also, I think things like `null` and `false` would get rendered in `content` which is unexpected compared to React?

Reply:
- React nodes (which are the renderable elements) are `ReactChild | ReactFragment | ReactPortal | boolean | null | undefined` (took from the TS definition).
- `ReactChild` accepts: `ReactElement | ReactText` and `ReactText` accepts strings or numbers.

BTW, yup, children don't accept more than one element! https://codesandbox.io/s/tippyjsreactpr96-lquyf

Reply:
I did reverted the children's type

Reply:
Ok that makes sense then! Yeah, I agree that `number` + an array of child types should work for `content`. Can you also update the `propTypes`, because there's still runtime errors for non-TS users?

Reply:
Done!